packages/http-client/src/remote-index-api.ts

```diff
@@ -17,19 +17,20 @@ export class RemoteIndexApi implements IndexApi {
   private readonly _collectionURL: URL
```

Comment:
should we add a `_countURL` here so we don't need to construct it inline each time?

Reply:
yup, to resolve the confusion